python/cuml/fil/fil.pyx

```diff
@@ -86,7 +86,7 @@ cdef class TreeliteModel():
     cdef ModelHandle get_handle(self):
         return self.handle
 
-    def __cdel__(self):
+    def __dealloc__(self):
```

Comment:
Excellent find, Saloni!

Reply:
`__cdel__` is also used for our `TreeliteModel`, in the same file. Would it also be possible to fix it (maybe in another pull request)? Note that only the host memory is allocated there, so it's not detected by the GPU memory leakage test.

Reply:
I believe I updated it in both places